src/enums/runtime.ts

```diff
@@ -20,16 +20,15 @@ export enum Runtime {
     Python311 = 'python-3.11',
     Python312 = 'python-3.12',
     Pythonml311 = 'python-ml-3.11',
-    Deno121 = 'deno-1.21',
-    Deno124 = 'deno-1.24',
-    Deno135 = 'deno-1.35',
```

Comment:
did we deprecate these?

Reply:
Yep: https://github.com/appwrite/runtimes/blob/main/src/Runtimes/Runtimes.php#L81C1-L83C132